CHANGELOG.md

```diff
@@ -9,3 +9,4 @@ Major changes since the last busboy release (0.31):
 * Error on non-number limit rather than ignoring (#7)
 * Dicer is now part of the busboy itself and not an external dependency (#14)
 * Tests were converted to Mocha (#11, #12, #22, #23)
+* Empty filenames of parts are handled as `undefined` (#26)

```

Comment:
can you also add entry to MIGRATING.md, explaining the breaking change?

Reply:
Yes, I can explain the breaking change.